cluster-sync/sync.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+export CDI_INSTALL_TIMEOUT=${CDI_INSTALL_TIMEOUT:-120}
+
+source ./cluster-up/hack/common.sh
+source ./cluster-up/cluster/${KUBEVIRT_PROVIDER}/provider.sh
+source ./cluster-sync/${KUBEVIRT_PROVIDER}/provider.sh
+
+# Install CDI
+install_cdi
```

Comment:
Maybe we could move cdi install to a function like we have for `kubevirt`.

Reply:
Well, it is a function same as kubevirt. You mean to add waiting to the function?

Reply:
yeah, I meant everything related to CDI